projects/hyperithm/index.js

```diff
@@ -0,0 +1,17 @@
+const { getLogs } = require("../helper/cache/getLogs");
```

Comment:
the first 3 lines of import can be removed from all these curators since it is not used?